docs/storage-abstraction.md

```diff
@@ -584,21 +523,26 @@ Scan scan =
         .limit(10)
         .build();
 
-// Execute the Scan operation
+// Execute the `Scan` operation.
 Scanner scanner = storage.scan(scan);
 ```
 
-Note that you can't specify clustering key boundaries and orderings in Scan without a partition key.
+{% capture notice--info %}
+**Note**
 
-#### Put operation
+You can't specify clustering-key boundaries and orderings in `Scan` without specifying a partition key.
+{% endcapture %}
 
-`Put` is an operation to put a record specified by a primary key.
-It behaves as an upsert operation: If it exists, it updates it; otherwise, it inserts it.
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
-You need to create a Put object first, and then you can execute it with the `storage.put()` method as follows:
+#### `Put` operation
```

Comment:
The structure for the following four sections in this doc are as follows:

```markdown
...
Put operation
  Conditions for Put
Delete operation
  Conditions for Delete
...
```

However, should this structure match the same structure as in `api-guide.md`, as shown below? Or do these sections need to be kept separate in this doc? 

```markdown
...
Put operation
Delete operation
Put and Delete with a condition
  Conditions for Put
  Conditions for Delete
...
```

Reply:
We can use the same structure as in the `api-guide.md`

Reply:
Got it. Thanks!